.github/actions/install/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Install Tools & Dependencies
+description: Installs pnpm, JS runtime & package dependencies
+
+inputs:
+  node-version:
+    description: 'Node version'
+    required: false
```

Comment:
Minor detail, but just in case: `required` defaults to `false`, so these 3 lines (7, 11, 15) could be removed. I put them in with the thought that it would be helpful to state this explicitly, but alternately there could be a comment between lines 4 & 5 saying something like "all of these are optional". And/or a comment could contain [a link to the docs](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs).

Reply:
I like having the default value here, even if it is implied.